MCB-Nucleo/Core/Inc/MCB.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * MCB.h
+ *
+ *  Created on: Jun. 1, 2023
+ *      Author: kcgro
+ */
+
+#ifndef INC_MCB_H_
+#define INC_MCB_H_
+
+#include "can.h"
+
+#define CAN_DATA_LENGTH 8
+#define ADC_POLL_DELAY 10
+#define SEND_MOTOR_COMMAND_DELAY 10
+#define ADC_DEADZONE 500
```

Comment:
Adding to magic number discussion in `freertos.c`, I would add a comment on all of these raw values. This would not be clear for a developer who was reading the codebase from new. What is the number representing - maybe comment it's for a specific type of pot. It might be even better to clarify 'ADC' with something more specific because there can be multiple ADC readings.

Reply:
Added comments for defines